app/presenters/groups_presenter.rb

```diff
@@ -1,5 +1,5 @@
 class GroupsPresenter
-  def initialize(group: group, current_user: current_user)
+  def initialize(group:, current_user:)
```

Comment:
Is this valid ruby syntax? Never seen a : without a corresponding hash value before.